docs/offboarding.rst

```diff
@@ -0,0 +1,162 @@
+Off-board Administrators and Journalists
+========================================
+
+When Journalists and SecureDrop Administrators leave your organization, it is important to off-board them from SecureDrop.
+
+**What you need:**
+
+- An *Admin Workstation*. Contact SecureDrop Support or follow our :doc:`guide to rebuilding an Admin Workstation <rebuild_admin>` if you do not have one.
+- An Admin account on the Journalist Interface
+
+.. important:: Note that, in particular for SecureDrop Administrators, additional measures may need to be considered if the user's departure is on unfavourable terms, and those measures will vary depending on the circumstances and your own internal incident response procedures. If you are in such a situation, consult the SecureDrop Support team or Digital Security Training team via the SecureDrop contact form, or email securedrop@freedom.press or a member of our team directly (found on the staff page). Note that, if you are in this situation, you may want to avoid reaching out to us via a ticket in our Support Portal, as that ticket will be visible to all Support Portal users at your organization.
+
+Off-boarding all users
+----------------------
+
+- `Inform the SecureDrop Support team`_ that the user's Support Portal account should be deactivated, and indicate if any new staff members should be added to the Support Portal..
```

Comment:
If you have an account on the support portal, ...

Reply:
IMO they should inform us of any user changes whether they have a support portal account or not, especially if the only support portal user leaves the organization and someone else is reading these docs trying to figure out how to take over admin duties. I have linked to the `getting-support` section here  in 1def8b9ee (and fixed the proper noun/typos!)-- what do you think?